addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -72,13 +72,14 @@ def test_mailing_list_merge(self):
         # This test ensure that the mailing lists are correctly merged and no
         # duplicates are appearing in C
 
-        result_list = self.env['mailing.list.merge'].create({
-            'src_list_ids': [(4, list_id) for list_id in [self.mailing_list_1.id, self.mailing_list_2.id]],
-            'dest_list_id': self.mailing_list_3.id,
-            'merge_options': 'existing',
-            'new_list_name': False,
-            'archive_src_lists': False,
-        }).action_mailing_lists_merge()
+        list_merge = self.env['mailing.list.merge'].with_context(active_ids=[self.mailing_list_1.id, self.mailing_list_2.id])
+
+        with Form(list_merge) as f:
```

Comment:
Actually why using a with statement at all ? Makes code complicated to read ... you could simply have
```
merge_form = Form() ...
```